### ISSUE TITLE
Sky stuck at daytime

### ISSUE THREAD

**Issue by avrdude-t** [eventhorizon5/skyscraper#13]
The sky never changes to nighttime, even if you use sky control to set it to night.
Also, whenever you press Ctrl+R to reload a bulding, this error appears:
![reload error](https://user-images.githubusercontent.com/44274249/206934100-4435a06d-989d-4560-bee0-24467b8a13a9.PNG)
If you press OK the building will still load fine though.

**Comment by eventhorizon5** [comment]
The sky currently doesn't work properly with the OpenGL3+ renderer plugin.  Are you using that, or the regular OpenGL one?  (you can find out if you open up the ogre.cfg file, it'll say at the top).  That error message was something I was trying to fix before, I'll look at it again.

**Comment by avrdude-t** [comment]
Yeah I was using OpenGL3+. It works fine on the normal OpenGL (except for the error).
However with further testing I discovered that the same problem also occures when using OpenGL ES 2.x, plus another error when loading a building:
![another one](https://user-images.githubusercontent.com/44274249/207419203-75bd0d7f-5515-4407-ae1a-2b886c4630dd.PNG)

**Comment by eventhorizon5** [comment]
It won't work with the OpenGL ES renderer either, is that error happening with that renderer?

**Comment by avrdude-t** [comment]
Yes, it only appears with ES. Also, in addition to appearing when reloading, the error also appears when intially loading a building.
